lib/onfido/webhook_event_verifier.rb

```diff
@@ -14,7 +14,7 @@ def read_payload(event_body, signature)
 
       raise(OnfidoInvalidSignatureError, "Invalid signature for webhook event") unless OpenSSL.secure_compare(signature, event_signature)
 
-      WebhookEvent.build_from_hash(JSON.parse(event_body)["payload"])
+      WebhookEvent.build_from_hash(JSON.parse(event_body))
```

Comment:
This is already modified on openapi side but not re-triggered here. Manually updated.